src/styles/index.scss

```diff
@@ -2,2 +1,19 @@
+@use './header.scss';
+@use './card-container';
+
+.page {
+  display: flex;
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+  scroll-behavior: smooth;
+}
+
+.body {
+  margin: 0;
+  place-content: center;
```

Comment:
The `place-content` property is a shorthand for `align-content` and `justify-content`. It seems like it's used here without a display property that would make it applicable. If you want to center the content, you might need to add `display: flex;` or `display: grid;` to this class.